packages/web-app-text-editor/src/App.vue

```diff
@@ -231,12 +231,19 @@ export default {
   }
 }
 </script>
-<style lang="scss" scoped>
+<style lang="scss">
 #text-editor-preview {
   max-height: 80vh;
   overflow-y: scroll;
 }
 #text-editor-input {
   resize: vertical;
 }
+.editor-wrapper-height {
```

Comment:
I'd prefer working with a full-height parent element and paddings/margins instead. Do you think that could work?

Reply:
f the height is set 100% and the text is longer than the height of the textarea (scrollbar showing), the upper buttons div gets stagely cut by me, the only workaround I found was to subtract the height of the upper buttons, margins or paddings don't fix it
![Screenshot from 2022-05-16 12-40-49](https://user-images.githubusercontent.com/7430156/168576286-12509325-07c1-45fe-8448-b9add1422e6f.png)